src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -278,6 +291,31 @@ export default {
       return publishRest.assetDownloadURI(this.identifier, this.version, asset_id);
     },
 
+    viewURI(asset_id, name) {
+      const file_ext = this.fileExtension(name);
+      if (file_ext === 'nii') {
+        return `https://bioimagesuiteweb.github.io/alphaapp/viewer.html?image=${publishRest.assetDownloadURI(this.identifier, this.version, asset_id)}`;
+      }
+      if (file_ext === 'nwb') {
+        return `http://nwbexplorer.opensourcebrain.org/nwbfile=${publishRest.assetDownloadURI(this.identifier, this.version, asset_id)}`;
+      }
+
```

Comment:
after https://github.com/bioimagesuiteweb/bisweb/issues/129 is addressed, we should also add `.tiff` into this "registry" (also with similar limits on the file size). ATM bisweb falls into "it is a nifti" if there is no file extension (as our endpoints and blob files have none), but IMHO should work whenever content-disposition is respected (where we do provide the extension)